package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/eth-sig-util",
-  "version": "5.0.1",
+  "version": "5.1.0",
```

Comment:
This will be done in the release branch
```suggestion
  "version": "5.0.1",
```

Reply:
Not sure what happened here... I didn't conciously make this change lol